popup/OpenClosed.md

```diff
@@ -0,0 +1,101 @@
+# Add a feature status as open or closed using date time fields
+
+This example shows how to add a custom custom AGO map viewer popup that tells if a park is open or closed. 
+
+## Use cases
+
+This is designed specifically for for the AGO map viewer popup configuration panel . These set of expressions allow you to represent the status of a feature, such as a store or a park, as either open or closed. 
+
+## Workflow
+
+This expression uses fields OPERHOURS and OPERDAYS and PHONE to identify wether the feature is currently open. 
+OPERHOURS is stored in the following formats:
+TIME - TIME (example: 08:00AM - 10:00PM)
+Daylight
+24 Hours/Day
+
+OPERDAYS is stored in the following formats:
+Day-Day (example: Sun-Sat)
+
+
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+To configure the script to your layer, you must replace the field variable to the name of the field containing the phases, set 'PHASE1' to the value of the corresponding phase value, and update the background-color to the HEX value of the color the phase. 
+
+## Expression Template
+
+For example, in a layer with a field called PROJPHASE with a domain for the 4 phases of the project (PreDesign, Design, Construction, and Closeout), you would replace this:
+
+```
+function Open(day,hours){
+
+var current_d = number(weekday(Now()));
+var current_t = Hour(Now());
+
+var CalcDay = when(day=="Other","Other",day=="Sun-Sat", "7", day=="Mon-Fri","5",day=="Sat-Sun","2","0");
```

Comment:
I suggest using `Decode` here to clean up this line so you're not using `day==` over and over again.

```suggestion
// Note I'm formatting the daysOpen value to lessen the risk of mismatching because of casing issues
var CalcDay = Decode( Lower(daysOpen), 
  "other", "other",
  "sun-sat", "7",
  "mon-fri", "5",
  "sat-sun", "2",
"0");
```